### ISSUE TITLE
Move the preprocessors out of pleeease and release them as plugins

### ISSUE THREAD

**Issue by yaycmyk** [iamvdo/pleeease#69]
Currently, every preprocessor is installed even if you're only using CSS. It would be good to split them out to reduce the installation burden and allow for community-developed ones.

**Comment by alexandruradulescu** [comment]
Agreed. I second this idea. This way, pleeease would have a smaller footprint, adding only the preprocessor one needs. Maybe in the way we could have cssnext also added :-D
